src/components/app/app.scss

```diff
@@ -8,7 +8,7 @@ body {
 }
 .app {
   width: 100%;
-  height: 100%;
+  height: 95%;
```

Comment:
Why this is 95% my friend?

Reply:
for some reason something is overflowing(at least in my browser) but i think we can fix that later